python/ql/lib/semmle/python/Flow.qll

```diff
@@ -507,18 +483,6 @@ class ImportStarNode extends ControlFlowNode {
 class SubscriptNode extends ControlFlowNode {
   SubscriptNode() { toAst(this) instanceof Subscript }
 
-  /**
-   * DEPRECATED: Use `getObject()` instead.
-   * This will be formally deprecated before the end 2018 and removed in 2019.
```

Comment:
Oh, I remember this comment. :laughing: 

It's a pity this didn't say "removed in 2020", seeing as we're still in March 2020 according to some people.